### PR TITLE
fix(transform property): apply transformation to gizmo

### LINKED ISSUE #7970: [Bug]: Transform not applying to objects in scene
If a model component's src is changed, the new scene will not update transforms according to moving the gizmo or updating the transform component properties: 
Screenshot from 2023-05-04 16-08-45

### PR DESCRIPTION
**Pull request by aditya-mitra**
## Summary

Fixed gizmo updating when the right-hand transform inputs are changed.

## References

closes #7970 

## Checklist
- [ ] If this PR is still a WIP, convert to a draft
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 **/node_modules/**
 /packages/server/upload/**
-/packages/projects/projects/**
+# /packages/projects/projects/**
```
Comment (HexaField) [inline .prettierignore:3]: Please revert

File: package.json [hunk 2]
```diff
@@ -135,12 +135,14 @@
...
     "app-root-path": "3.1.0",
+    "bops": "^1.0.1",
     "cli": "1.0.1",
     "dotenv": "16.0.3",
     "fix-esm": "^1.0.1",
     "patch-package": "^6.5.1",
     "three": "0.150.1",
     "ts-node": "10.9.1",
-    "typescript": "5.0.2"
+    "typescript": "5.0.2",
```
Comment (HexaField) [inline package.json:145]: Please revert

File: package.json [hunk 3]
```diff
@@ -135,12 +135,14 @@
     "@feathersjs/schema": "5.0.5",
     "@feathersjs/typebox": "5.0.5",
     "app-root-path": "3.1.0",
+    "bops": "^1.0.1",
```
Comment (HexaField) [inline package.json:138]: Please revert

**Review by HexaField — CHANGES_REQUESTED** [review]

**Comment by aditya-mitra** [comment]
Sorry. Accidentally committed the files with `git add .`

**Review by HexaField — CHANGES_REQUESTED** [review]
position and rotation looks good, but scaling with either the gizmo nor vector inputs do not show on the gizmo at all

**Pull request merged**